src/js/utils/language.js

```diff
@@ -50,7 +50,10 @@ function extractLanguage(doc) {
 export function getLanguage() {
     let language = extractLanguage(document);
     if (!language && isIframe()) {
-        language = extractLanguage(window.top.document);
+        try {
+            // Exception is thrown if iFrame is on a different domain name.
+            language = extractLanguage(window.top.document);
+        } catch (e) {/* ignore */}
     }
     return language || navigator.language || navigator.browserLanguage || navigator.userLanguage || navigator.systemLanguage || 'en';
```

Comment:
Why are we falling back to `navigator.language` or `navigator.browserLanguage`? These represent the preferred language of the user which I though we did not want to use.

Reply:
IE 11 supports `navigator.language`. If we want the user's preferred language, that is all we need to check. `navigator.browserLanguage || navigator.userLanguage || navigator.systemLanguage` are not needed.